src/components/NavigationTree/NavigationTree.tsx

```diff
@@ -34,6 +34,7 @@ export function NavigationTree({
     rootState: partialRootState,
     fetchPath,
     getActions,
+    getOptions,
```

Comment:
I don't like `options` and `getOptions` naming for `JSX` and function, that returns `JSX`. Maybe `additionalControls` and `renderAdditionalControls` or `additionalNodeElements` and `renderAdditionalNodeElements` or something like that?